pints/_log_priors.py

```diff
@@ -95,6 +97,93 @@ def sample(self, n=1):
         return np.random.beta(self._a, self._b, size=(n, 1))
 
 
+class BinomialLogPrior(pints.LogPrior):
+    r"""
+    Defines a binomial (log) prior with given number of trials parameter
+    ``trials`` and trial success probability parameter ``prob`` with
+    pdf
+
+    .. math::
+        f(x|\text{trials}, \text{prob}) = {\text{trials} \choose x}\;
+        \text{prob}^{x}\;(1-\text{prob})^(\text{trials}-x).
+
+    A random variable :math:`X` distributed according to this pdf has
+    expectation
+
+    .. math::
+        \mathrm{E}(X)=\text{trials} \; \text{prob}.
+
+    For example, to create a prior with ``trials=10`` and ``prob=0.5``
+    use::
+
+        p = pints.BinomialLogPrior(10, 0.5)
+
+    Extends :class:`LogPrior`.
+    """
+
+    def __init__(self, trials, prob):
+        # Parse input arguments
+        self._trials = float(trials)
+        self._prob = float(prob)
+
+        # Validate inputs
+        if not self._trials.is_integer():
+            raise TypeError('Count parameter "trials" must be \
+                integer.')
+        if self._trials <= 0:
+            raise ValueError('Count parameter "trials" must be \
+                positive.')
+        if not ((self._prob > 0) and (self._prob < 1)):
+            raise ValueError('Probability parameter "prob" must be \
+                between 0 and 1.')
+
+        # Cache constant
+        self._log_prob = np.log(self._prob)
+        self._log_opo_prob = np.log(1 - self._prob)
+
+    def __call__(self, x):
+        if x[0] < 0.0 or x[0] > self._trials or not float(x[0]).is_integer():
+            return -np.inf
+        else:
+            return np.log(
+                scipy.special.comb(self._trials, x[0])) +\
+                self._log_opo_prob * (self._trials - x[0]) +\
```

Comment:
```suggestion
                scipy.special.comb(self._trials, x[0])) +
                self._log_opo_prob * (self._trials - x[0]) +
```
you don't need the `\` within parentheses

So it's
```
x = a \
    + b
```
or
```
x = (a
    + b)
```